resources/views/pages-legacy/reorderSiteAwards.blade.php

```diff
@@ -108,6 +82,9 @@ function postAllAwardsDisplayOrder(awards) {
 
             <div class="flex w-full items-center justify-between mt-3 mb-6">
                 <div class="flex items-center gap-x-1">
+                    <div class="ml-auto mr-4">
+                         <button onclick='reorderSiteAwards.handleResetOrder()' class="btn text-3xs py-0.6">Reset Order</button>
```

Comment:
a11y: It only now occurs to me, in the button's present form, it's a destructive action.

We may want to add a "btn-danger" class to this element.

It may also be desirable to swap its position with the checkbox in case anyone reaches for the checkbox out of muscle memory.